dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -25,23 +25,7 @@
 .script_text
   = text_area_tag('level[dsl_text]', @level.dsl_text || (@level.dsl_default))
 
-- if @level.is_a?(External)
-  .field
-    = f.label :icon_type, 'Icon'
-    %p
-      Override the icon that shows up for this level
-      %div
-        Default:
-        %i.fa.fa-file-text
-        Stop:
-        %i.fa.fa-stop-circle
-        Map:
-        %i.fa.fa-map
-
-    = f.select :icon_type, options_for_select(["Default", "Stop", "Map"], @level.icon_type)
-
-- if [ContractMatch, Match, Multi, TextMatch, External].include? @level.class
-
+- if @level.supports_markdown?
```

Comment:
This change for supports_markdown isn't strictly necessary, but I think this is a cleaner approach than having a list of class names.